crates/sp-domains/src/lib.rs

```diff
@@ -498,6 +498,25 @@ where
     }
 }
 
+#[derive(Serialize, Deserialize)]
+pub struct GenesisDomainRuntime {
+    pub name: Vec<u8>,
+    pub runtime_type: RuntimeType,
+    pub code: Vec<u8>,
+}
+
+/// Types of runtime pallet domains currently supports
+#[derive(
+    TypeInfo, Debug, Default, Encode, Decode, Clone, PartialEq, Eq, Serialize, Deserialize,
+)]
+pub enum RuntimeType {
```

Comment:
What's the difference between the runtime name and runtime type? It's defined in the spec, but there isn't more explanation on the use case of runtime type. The runtime name seems to be not necessary now.

Reply:
Runtime type is the type of the runtime where as name could be custom we could give. I think this could be used as metadata. For example, a runtime of type EVM with different combinations of pallets. 
This is a stretch but lets keep it until we really have no use for it